browser/main/Detail/ToggleStackDirectionButton.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import CSSModules from 'browser/lib/CSSModules'
+import styles from './ToggleStackDirectionButton.styl'
+import i18n from 'browser/lib/i18n'
+
+const ToggleStackDirectionButton = ({
+  onClick, isStacking
+}) => (
+  <button styleName='control-splitPanelDirection' onClick={() => onClick(!isStacking)}>
+    <img styleName='iconInfo' src={isStacking ? '../resources/icon/icon-panel-split-vertical.svg' : '../resources/icon/icon-panel-split-horizontal.svg'} />
+    <span lang={i18n.locale} styleName='tooltip'>{
+          isStacking ? i18n.__('Split Panels Horizontally') : i18n.__('Split Panels Vertically')
+
```

Comment:
Can  you adjust the code format here? I think you should separate the `src` for image and the content for `span` tag into smaller variables for better readability

Reply:
I have refactored the image src and the span text into smaller vars.